client/src/components/reviewCriteria.jsx

```diff
@@ -1,16 +1,89 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+
+import StarRatings from 'react-star-ratings';
+
 
 export default class ReviewCriteria extends React.Component {
   constructor(props) {
     super(props);
   }
 
   render() {
-    return (
-      <div>
-
-      </div>
-    );
+    if (this.props.reviews.length) {
+      return (
+        <div id="revCriteriaContainer">
+          <div id="line1">
+            <div>Accuracy</div>
+            <div id="AccStars">
+              <StarRatings
```

Comment:
Just a suggestion, since your StarRatings components each take the same attributes, if you _wanted_ to, you could probably split this into smaller components for cleaner code that just take your props, something along these lines:

`<Stars divtitle="Accuracy" ratings={Math.round(this.props.criteria.accuracy)} />
<Stars divtitle="Location" ratings={Math.round(this.props.criteria.location)} />
<Stars divtitle="Communication" ratings={Math.round(this.props.criteria.communication)} />
<Stars divtitle="Check-in" ratings={Math.round(this.props.criteria.checkin)} />
<Stars divtitle="Cleanliness" ratings={Math.round(this.props.criteria.cleanliness)} />
<Stars divtitle="Value" ratings={Math.round(this.props.criteria.value)} />`